test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -1904,3 +1876,22 @@ func checkCpuGovernorsAndResumeLatency(cpus []int, targetNode *corev1.Node, pm_q
 	}
 	return nil
 }
+
+// checkHardwareCapability Checks if test is run on baremetal worker
+func checkHardwareCapability(workerRTNodes []corev1.Node) {
+	for _, node := range workerRTNodes {
+		numaInfo, err := nodes.GetNumaNodes(&node)
+		Expect(err).ToNot(HaveOccurred())
+		if len(numaInfo) < 2 {
+			Skip(fmt.Sprintf("This test need 2 NUMA nodes.The number of NUMA nodes on node %s < 2", node.Name))
+		}
+		// Additional check so that test gets skipped on vm with fake numa
+		onlineCPUCount, err := nodes.ExecCommandOnNode([]string{"nproc", "--all"}, &node)
+		Expect(err).ToNot(HaveOccurred())
+		onlineCPUInt, err := strconv.Atoi(onlineCPUCount)
+		Expect(err).ToNot(HaveOccurred())
+		if onlineCPUInt < 80 {
```

Comment:
Do the test systems all really have 80 or more cpus? That seems high.

Reply:
The systems that we work  like Dell R740 has 80cpus

Reply:
okay, i will reduce it to 32 cpus

Reply:
addressed this in the latest commit